dnf-plugins-core.spec

```diff
@@ -579,6 +588,7 @@ PYTHONPATH=./plugins nosetests-%{python3_version} -s tests/
 %{_bindir}/yum-config-manager
 %{_bindir}/yum-debug-dump
 %{_bindir}/yum-debug-restore
+%{_bindir}/yum-groups-manager
```

Comment:
Do we really want to add this yum compatibility "bloat" at this stage? I'd only do it if explicitly requested, and from the bug it appears it is not (_"They requested to have yum-groups-manager (**or a similar capability**) ported over to dnf."_).

Reply:
It's part of separate yum compatibility package (yum-utils on RHEL, dnf-utils on Fedoras). We do this for all other yum tools like reposync, repoquery... and I think it will be more confusing for the user to have different tools handled in different way.

Reply:
Ok, I don't know, my point is we want users to stop using these, they're backwards compatibility only. Ideally new users shouldn't be confused as they shouldn't be using them. Right? I don't think they're marked as deprecated, except for that first line of text in `doc/dnf-utils.rst` saying they're for backwards compatibility? (meaning the users don't necessarily get the right message)

I get your point about consistency, but i don't see that much value in adding in essence deprecated compatibility layer to maintain the consistency...

That said I'm not going to insist if you insist :slightly_smiling_face: I'd just rather be removing these instead of adding them.